apps/login/app/passkeys/route.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  createPasskeyRegistrationLink,
+  getSession,
+  registerPasskey,
+  server,
+} from "#/lib/zitadel";
+import { getSessionCookieById } from "#/utils/cookies";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(request: NextRequest) {
+  const body = await request.json();
+  if (body) {
+    const { sessionId } = body;
+
+    const sessionCookie = await getSessionCookieById(sessionId);
+
+    const session = await getSession(
+      server,
+      sessionCookie.id,
+      sessionCookie.token
+    );
+
+    const domain: string = request.nextUrl.hostname;
+
+    const userId = session?.session?.factors?.user?.id;
+
+    if (userId) {
+      return createPasskeyRegistrationLink(userId)
+        .then((resp) => {
+          const code = resp.code;
+          return registerPasskey(userId, code, domain).then((resp) => {
+            return NextResponse.json(resp);
+          });
+        })
+        .catch((error) => {
+          console.error("error on creating passkey registration link");
```

Comment:
I think we don't want to log to stdout, right?

Reply:
i left it too, otherwise you dont see it in the browser console

Reply:
These are basically access logs, that we should be able to query from the LB, right? Otherwise we should configure it globally, IMO, with an interceptor or so.

Reply:
yes